src/styles/left-nav.css

```diff
@@ -0,0 +1,21 @@
+.ds-c-vertical-nav > .ds-c-vertical-nav__item {
+  border-top: none;
+}
+
+.ds-c-vertical-nav a.ds-c-vertical-nav__label:visited {
+  /* QPP "Navy" */
+  color: #125496;
+}
+
+.ds-c-vertical-nav a.ds-c-vertical-nav__label {
+  border: none;
+}
+
+.ds-c-vertical-nav a.ds-c-vertical-nav__label--current {
+  /* QPP "Medium Teal" */
+  border-left: 4px solid #209fa6;
+  font-weight: normal;
+  background: #f1f1f1;
+  /* QPP "Navy" */
+  color: #125496;
+}
```

Comment:
these are essentially design system overrides too, but specific to the `LeftNav` component. if we add a `left-nav` class like in the comment above, these styles could be written as:

```css
.left-nav.ds-c-vertical-nav > .ds-c-vertical-nav__item {
  border-top: none;
}

.left-nav.ds-c-vertical-nav a.ds-c-vertical-nav__label:visited {
  /* QPP "Navy" */
  color: #125496;
}

.left-nav.ds-c-vertical-nav a.ds-c-vertical-nav__label {
  border: none;
}
```
and would be specific to the `LeftNav` component while still retaining the necessary specificity.

Reply:
cc @sawyerh is this a reasonable way to modify design system rules in a scoped way?

Reply:
I think a less specific selector would also work (design system selectors generally have very low specificity):

```css
.left-nav .ds-c-vertical-nav__label:visited {
  /* QPP "Navy" */
  color: #125496;
}

.left-nav .ds-c-vertical-nav__label {
  border: none;
}
```